Cockpit 319 (#745)
cockpit-project/cockpit-project.github.io: _posts/2024-06-26-cockpit-319.md

---
title: Cockpit 319
author:
date: '2024-06-26'
tags: cockpit, podman, files
slug: cockpit-319
category: release
summary: 'btrfs snapshots, podman image pull, files bookmark support'
---

Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
We release regularly.

Here are the release notes from Cockpit 319, cockpit-machines 315, cockpit-podman 90, and cockpit-files 2:

> ## Btrfs snapshots

Reviewer: ```suggestion
## Storage: Btrfs snapshots
```

Ah! Forgot the prefix! Whoops!

> Snapshots of a subvolume are now listed in the subvolume detail page.

Reviewer: ```suggestion
Subvolume snapshots are now listed on subvolume detail pages.
```

![screenshot of ## show snapshots of a subvolume in the detail view]({{ site.baseurl }}/images/319-show-snapshots-of-a-subvolume-in-the-detail-view.png)
{:.screenshot}

> The snapshot origin is shown in snapshot detail view.

Reviewer: ```suggestion
Snapshot origins are shown in the snapshots detail view.
```

![screenshot of ## show snapshots of a subvolume in the detail view]({{ site.baseurl }}/images/319-show-snapshots-of-a-subvolume-in-the-detail-view-2.png)
{:.screenshot.left}

## Podman: Add image pull action

> The new `Pull` option in the image context menu updates the existing image to the latest version on its registry.

Reviewer: ```suggestion
Use the "Pull" option in an image's context menu to update it to the latest available version.
```

![screenshot of add image pull action]({{ site.baseurl }}/images/319-podman-add-image-pull-action.png)
{:.screenshot.left}

## Files: Bookmark support

> The current working directory can be bookmarked for easy navigation.

Reviewer: ```suggestion
Navigate to saved locations or add the current directory to bookmarks using the new bookmarks button.
```
